runtime/src/lib.rs

```diff
@@ -10,6 +10,7 @@ mod wasm_glue;
 
 mod write;
 pub use write::DiplomatWrite;
+pub use write::{diplomat_buffer_write_create, diplomat_buffer_write_destroy};
```

Comment:
this feels wrong. I would propose just adding a method to the example lib that uses a String to write to.